TERMINAL LIST
Type:      File
Name:      f1
Original:  <dir>/r3/f1
Created:   2020-05-31 12:00:00 
Entries:      1
Total Files:  1
Log size:     0.00B
Size:         10.00B 


EXCEPTION RESTORE
Encountered exception(s)

- Path not found: bad file
CallStack (from HasCallStack):
  foldKeys, called at src/SafeRm/Data/Index.hs:137:9 in safe-rm-0.1-inplace:SafeRm.Data.Index
  searchIndex, called at src/SafeRm.hs:267:32 in safe-rm-0.1-inplace:SafeRm
  restore, called at src/SafeRm/Runner.hs:136:24 in safe-rm-0.1-inplace:SafeRm.Runner
  runCmd, called at test/functional/Functional/Prelude.hs:215:36 in main:Functional.Prelude


CallStack (from HasCallStack):
  throwCS, called at src/SafeRm.hs:291:5 in safe-rm-0.1-inplace:SafeRm
  restore, called at src/SafeRm/Runner.hs:136:24 in safe-rm-0.1-inplace:SafeRm.Runner
  runCmd, called at test/functional/Functional/Prelude.hs:215:36 in main:Functional.Prelude

LOGS RESTORE
[2020-05-31 12:00:00][functional.restore][Debug] Trash home: <dir>/r3/.trash
[2020-05-31 12:00:00][functional.restore.readIndex][Debug] Index path: <dir>/r3/.trash/.index.csv
[2020-05-31 12:00:00][functional.restore.readIndex.readIndexWithFold][Debug] Found: MkPathData {pathType = PathTypeFile, fileName = MkPathI {unPathI = "f1"}, originalPath = MkPathI {unPathI = "<dir>/r3/f1"}, created = MkTimestamp {unTimestamp = 2020-05-31 12:00:00}}
[2020-05-31 12:00:00][functional.restore][Info] Restored: 
[2020-05-31 12:00:00][functional][Error] Encountered exception(s)

- Path not found: bad file
CallStack (from HasCallStack):
  foldKeys, called at src/SafeRm/Data/Index.hs:137:9 in safe-rm-0.1-inplace:SafeRm.Data.Index
  searchIndex, called at src/SafeRm.hs:267:32 in safe-rm-0.1-inplace:SafeRm
  restore, called at src/SafeRm/Runner.hs:136:24 in safe-rm-0.1-inplace:SafeRm.Runner
  runCmd, called at test/functional/Functional/Prelude.hs:215:36 in main:Functional.Prelude


CallStack (from HasCallStack):
  throwCS, called at src/SafeRm.hs:291:5 in safe-rm-0.1-inplace:SafeRm
  restore, called at src/SafeRm/Runner.hs:136:24 in safe-rm-0.1-inplace:SafeRm.Runner
  runCmd, called at test/functional/Functional/Prelude.hs:215:36 in main:Functional.Prelude


